internal/hardware/hardware.go

```diff
@@ -89,3 +89,27 @@ func MapHardware(hardware *models.HardwareInfo) *v1alpha1.Hardware {
 	}
 	return &hw
 }
+
+func MapLabels(hardware *models.HardwareInfo) map[string]string {
+	if hardware == nil {
+		return nil
+	}
+	var labels = make(map[string]string)
+
+	labels["feature.device/hostname"] = hardware.Hostname
+
+	cpu := hardware.CPU
+	if cpu != nil {
+		labels[fmt.Sprintf("feature.device/cpu.%s", cpu.Architecture)] = "true"
```

Comment:
Was there a reason for selecting a label name to start with `feature.device/`?
If there is no strong reason for it, perhaps we should have a common prefix for all of operator's labels (and annotation)

Reply:
I tried to follow convention used [here](https://github.com/kubernetes-sigs/node-feature-discovery). We can change it. It is up to us to define common prefix.

Reply:
For some reason k8s complains about `/`. I updated the prefix. Please let me know whether it is something we would like to use.

Reply:
@pkliczewski I do not understand what is the reason to follow what others are doing. We know what to do and what makes sense. I don't see any reasoning for this pattern.